src/main/cpp/src/utilities.hpp

```diff
@@ -39,4 +81,41 @@ std::unique_ptr<rmm::device_buffer> bitmask_bitwise_or(
   rmm::cuda_stream_view stream      = cudf::get_default_stream(),
   rmm::device_async_resource_ref mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Throws an error with the row index if has any row is invalid for a unary operation.
+ * If the input is not null and the result is null, it means the row is invalid.
+ * @param input The input column view.
+ * @param result The result column view.
+ */
+void throw_row_error_if_has(cudf::column_view const& input,
```

Comment:
How about `throw_row_error_if_any`? Since `any` is more aligned with other standard APIs than `_has`.